mitodl/ocw-hugo-themes
# Update President's message on About OCW page

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3276 and closes https://github.com/mitodl/hq/issues/3543.

### Description (What does it do?)
This PR updates the President's Message on the About OCW page with an image of and message from MIT President Sally Kornbluth.

### How can this be tested?
Run `yarn start www`, and then navigate to `http://localhost:3000/about`. Verify that the President's Message section has been updated properly.
